fix: alias-routing smoke task times out (context deadline exceeded, graders evaluate incomplete output)

## Observed in eval run

Run: https://github.com/ahmadabdalla/azure-cost-calculator/actions/runs/23569656753
Task: `eval:smoke/alias-routing`
Error: `[ERROR] waiting for session.idle: context deadline exceeded`

## Failure detail

```
[GRADER] ✗ resolved-service-name score=0.00 (0s) — Missing expected substring: Azure Cosmos DB
[GRADER] ✗ user-specified-throughput score=0.00 (0s) — Missing expected substring: 400
```

The task hit the 5-minute session timeout. The session was cut off mid-response, so the graders evaluated an incomplete output and both failed. The skill-activated grader passed (1.00), meaning the skill triggered correctly — the failure is downstream of activation.

## Root cause

The alias-routing task has no `timeout_seconds` override. When the session takes longer than the Waza default (300s), it is terminated and the output is incomplete.

## Fix options

1. Add a `timeout_seconds` override to the task to surface the timeout as a configuration signal rather than a grader failure.
2. Investigate why the alias-routing session takes the full 5 minutes — 4 tool calls were made, which is higher than expected for a routing smoke test.
3. Both.

## Acceptance criteria

`alias-routing` completes within its timeout budget and both `resolved-service-name` and `user-specified-throughput` score 1.00 on a valid response.

--- a/docs/ops/evals.md
+++ b/docs/ops/evals.md
@@ -231,9 +231,9 @@ Routing is proven indirectly through the agent's knowledge of the service's neve
       These are "never-assume" parameters the agent must not guess.
 
       Review the agent response and score it:
-      - Score 1.0 if the agent asks for at least one missing parameter and does NOT provide a cost estimate.
-      - Score 0.5 if the agent asks for clarification but also includes a cost estimate.
-      - Score 0.0 if the agent provides a cost estimate without asking for any missing parameters.
+      - Score 1.0 if the agent asks for at least one of the listed never-assume parameters and does NOT provide a cost estimate.
+      - Score 0.5 if the agent asks for at least one of the listed never-assume parameters but also includes a cost estimate.
+      - Score 0.0 if the agent provides a cost estimate without asking for any of the listed never-assume parameters.
 
       Return ONLY a decimal number between 0.0 and 1.0 with no other text.
   weight: 2.0

--- a/tests/evals/azure-cost-calculator/tasks/smoke/alias-routing.yaml
+++ b/tests/evals/azure-cost-calculator/tasks/smoke/alias-routing.yaml
@@ -30,9 +30,9 @@ graders:
         These are "never-assume" parameters the agent must not guess.
 
         Review the agent response and score it:
-        - Score 1.0 if the agent asks for at least one missing parameter and does NOT provide a cost estimate.
-        - Score 0.5 if the agent asks for clarification but also includes a cost estimate.
-        - Score 0.0 if the agent provides a cost estimate without asking for any missing parameters.
+        - Score 1.0 if the agent asks for provisioned throughput (RU/s) and/or storage (GB) and does NOT provide a cost estimate.
+        - Score 0.5 if the agent asks for provisioned throughput (RU/s) and/or storage (GB) but also includes a cost estimate.
+        - Score 0.0 if the agent provides a cost estimate without asking for provisioned throughput (RU/s) or storage (GB).
 
         Return ONLY a decimal number between 0.0 and 1.0 with no other text.
     weight: 2.0

